sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2064,7 +2064,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
 
   test("show functions") {
     withUserDefinedFunction("add_one" -> true) {
-      val numFunctions = FunctionRegistry.functionSet.size.toLong
+      val numFunctions = FunctionRegistry.functionSet.size.toLong + 4L
```

Comment:
ah missed this one. We should use `FunctionsCommand.virtualOperators.length` to be future-proof.

Reply:
> ah missed this one. We should use `FunctionsCommand.virtualOperators.length` to be future-proof.

Good ideal, make it more clear. Done